plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/_launcher_util.py

```diff
@@ -117,7 +118,12 @@ def ray_down(yaml_path: str) -> None:
 
 
 def ray_up(yaml_path: str) -> None:
-    _run_command(["ray", "up", "-y", yaml_path])
+    print(f"RAY UP : {yaml_path}")
+    out, err = _run_command(["ray", "up", "-y", yaml_path])
+    print("OUT")
```

Comment:
Why are you not using the logger?

Reply:
fixed. I was planning on cleaning this up :)